### ISSUE TITLE
Pulling an players stats from an individual season

### ISSUE THREAD

**Issue by newenglandfall** [robhowley/nhlscrapi#17]
Is it possible to pull a players individual stats from a season or multiple seasons?

**Comment by robhowley** [comment]
No, sorry. Never got around to adding support for the player profile pages
on the site. If you'd like to add it, I'd love to approve the pr.
